BuildAndTest.cmd

```diff
@@ -5,6 +5,12 @@ SETLOCAL
 @REM create a nuget package for the SARIF SDK) so must opt-in
 @REM %~dp0.nuget\NuGet.exe update -self
 
+call BeforeBuild.cmd
```

Comment:
Not that it matters alot, but let's put this down below all those `echo` commands.. that is, let's put this new code in the same place you put the old code. It just makes the GitHub diff easier to read.